samples/maven-sample/.idea/codeStyles/Project.xml

```diff
@@ -0,0 +1,16 @@
+<component name="ProjectCodeStyleConfiguration">
```

Comment:
why do we have the folder .idea in each module? Usually, we have only one folder .idea

Reply:
yes, but if you open the submodule, the settings will go wrong, so I decided to add the same settings to the submodules. I'll fix it now